forced_photometry/multiband_photometry.ipynb

```diff
@@ -1131,7 +1099,7 @@
    "outputs": [],
    "source": [
     "#nway calling sequence\n",
-    "!nway.py '../data/Chandra/COSMOS_chandra.fits' :ERROR_RADIUS '../data/multiband_phot.fits' 0.1 --out=../data/Chandra/chandra_multiband.fits --radius 15 --prior-completeness 0.9"
+    "!/home/jovyan/.local/bin/nway.py '../data/Chandra/COSMOS_chandra.fits' :ERROR_RADIUS '../data/multiband_phot.fits' 0.1 --out=../data/Chandra/chandra_multiband.fits --radius 15 --prior-completeness 0.9"
```

Comment:
not a big deal, but it would be much nicer if the python software could be used as a library rather than a cmdline tool. I looked at it's repo, not obvious whether it could be done easily.

Reply:
I agree, it is a bit strange to make a command line call. I don't remember the details because it has been a long time since I worked with nway, but I think for now, since it does work, we should leave it as is.